openlibrary/catalog/get_ia.py

```diff
@@ -226,33 +235,3 @@ def marc_formats(identifier, host=None, path=None):
         if all(has.values()):
             break
     return has
-
-def get_from_local(locator):
```

Comment:
I am fine to delete these functions if they are not needed if it would be better to them then see #3053.